pkg/kubernetes/strategicmerge/v1alpha1/strategicmerge.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
```

Comment:
We are not using any `spec` here. Why are we having `v1alpha1` a version here? We should avoid versioning here IMO.